js/renovation/widget.tsx

```diff
@@ -55,7 +57,7 @@ const getAttributes = ({ elementAttr, accessKey }) => {
     return attrs;
 };
 
-const getCssClasses = (model: Partial<Widget>) => {
+const getCssClasses = (model: Partial<Widget> & Partial<WidgetInput>) => {
```

Comment:
Can we use just `model: Partial<Widget>` here?

Reply:
No, we pass an object that contains a mix of props and internal states
https://github.com/DevExpress/DevExtreme/blob/2177399ec14e7d4a4b455bfcfcb196535353c8cf/js/renovation/widget.tsx#L107

Reply:
As I understand, we combine internal state and input props into the Widget class.

Reply:
We need using `this.props.member` to access to a member of ComponentInput
https://github.com/DevExpress/DevExtreme/blob/2177399ec14e7d4a4b455bfcfcb196535353c8cf/js/renovation/widget.tsx#L82